api/public/cerbos/response/v1/response.proto

```diff
@@ -34,6 +34,7 @@ message PlanResourcesResponse {
       description: "Policy scope that matched to produce this effect."
       example: "\"acme.corp.base\""
     }];
+    map<string, string> matched_scope_per_action = 3 [(grpc.gateway.protoc_gen_openapiv2.options.openapiv2_field) = {description: "Matched scope for each action."}];
```

Comment:
```suggestion
    map<string, string> matched_scopes = 3 [(grpc.gateway.protoc_gen_openapiv2.options.openapiv2_field) = {description: "Matched scope for each action."}];
```

I think we should populate this field even when it's a singular `action` in the request. Then we can slowly deprecate the `matched_scope` field and fully transition to this new field at some point. In fact, in the follow-up task of updating the SDKs, we should start using the new fields instead of the old ones.